src/fcopy.c

```diff
@@ -0,0 +1,60 @@
+/* Copyright 2018. Damien Nguyen
```

Comment:
Hi Damien,
would changing 'copy' to also work for simple copies also be OK for your use case instead of adding a new command? (I already have such a patch locally)?

Reply:
Yes that would work. If you prefer not to merge these changes, I could add this functionality outside of BART when I embed it.

Reply:
Ok, but I then also add the necessary change to 'copy'. It is useful functionality, but I rather having another copy command.